doc/Config_Tutorials/Pbo_Creation_Dependence_Includes.md

```diff
@@ -0,0 +1,116 @@
+<!--
+I reccomend viewing this document with the extension Markdown Preview Enhanced (https://marketplace.visualstudio.com/items?itemName=shd101wyy.markdown-preview-enhanced). Once installed to VS Code you can see a html preview by clicking the pane on the right side of VS Code that looks like an open book with a magnifying glass. This will greatly enchance readability.
+-->
+
+# Initialization and #Include
+<p>In <a href="./Texture_Swaps.md">Texture Swaps</a> we mentioned that when pulling from another mod you need to both initialize a class (sometimes called defining it), and also declare the mod it comes from as a dependacy for our .pbo. This document will cover both how to do that and how #include works so that you know where to initialize a class. Note that this meaning of define is not the same as #define which is something else entirely.</P>
+
+## Mod Decleration
+<p>The following code block is a complete example of how to define a mod. It starts with '<i>class CfgPathces</i> and then has an internal class which is the mod's true name according to Arma. It is a good practice to name this the same as your folder structure so that other people can list your mod as a requirement without unpacking it. Many mods do not do this however so to find a mod's true name you need to unpack it and look in the .cpp file.</P>

```

Comment:
```suggestion
<p>The following code block is a complete example of how to define a mod. It starts with 'class CfgPathces` and then has an internal class which is the mod's true name according to Arma. It is a good practice to name this the same as your folder structure so that other people can list your mod as a requirement without unpacking it. Many mods do not do this however so to find a mod's true name you need to unpack it and look in the .cpp file.</P>

```

Mini code snippet